client/api/assessments.js

```diff
@@ -0,0 +1,12 @@
+import request from 'superagent'
+
+const URL = 'http:localhost:3000/v1/api'
```

Comment:
I believe this api should be `/api/v1/assessments` as stated in server.js

Reply:
Sorry you are right. Has been changed